docs/bugs_and_glitches.md

```diff
@@ -2065,6 +2066,24 @@ Most of the NPCs in [maps/NationalParkBugContest.asm](https://github.com/pret/po
 (The use of `SPRITE_ROCKER` instead of `SPRITE_COOLTRAINER_M` for `COOLTRAINERM NICK` may also be an intentional reference to the player's brother from the [Space World '97 beta](https://github.com/pret/pokegold-spaceworld).)
 
 
+### `BattleAnim_Tackle` reaches horizontal sprite limit
+
+`BattleAnim_Tackle` in Pokémon Crystal reaches the horiztonal sprite limit due to two rows of the enemy's feet being copied from background tiles to sprites. This fix restores the animation to only copy one row like in Pokémon Gold and Silver.
+
```

Comment:
This could be less verbose too; seeing the fix explains what it's doing.
```suggestion
`BattleAnim_Tackle` in Pokémon Crystal reaches the horizontal sprite limit due to two rows of the enemy's feet being copied from background tiles to sprites. This fix restores the animation to only copy one row like in Pokémon Gold and Silver.
```